### DISCUSSION TITLE
Metadata for cloud-bolt is incorrect (metadata does not include icon history)

### DISCUSSION THREAD

**Discussion by iJunkie22** [FortAwesome/Font-Awesome#20913] [Feature Requests]
The metadata for `cloud-bolt` indicates that the icon existed in `v5.5.0`, but the `.svg` files do not exist in *any* of `5.x`. (It is untested whether they exist in the `5.x` fonts).

https://github.com/FortAwesome/Font-Awesome/blob/deeea78c52bfe00b6e251ffddccf5570d5fdb05e/metadata/icons.yml#L9671

Correcting this error requires changing `icons.yml` and `icons.json`.

This error also shows up on the website (see screenshot).

![image](https://github.com/FortAwesome/Font-Awesome/assets/6920970/4d853d23-2726-4cb7-85af-a48feb222946)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

`fa-cloud-bolt` exists under 5.5.0 Pro with the name `thunderstorm`

https://fontawesome.com/v5/icons/thunderstorm?f=classic&s=solid

It is still available with the `thunderstorm` alias in FA 6 and I guess it has been moved to Free Icons together with the 6.1.0 release including Humanitarian icons

I understand that this is confusing, but I guess that at the moment we do not have the amount of metadata to generate "history log" of icons, like:
> - "Released in 5.5.0 in Pro Set"
> - "Released in 6.1.0 in Free Set"
> - "Updated in x "
> - "Renamed to x "

/cc @robmadole

**Comment by robmadole** [comment]
Our current metadata is not this rich, you are correct @tagliala .

In the future we could make this a richer experience...
